stanford/roles/deployment/defaults/main.yml

```diff
@@ -5,4 +5,14 @@ DEPLOYMENT_NAME: "{{ COMMON_DEPLOYMENT }}-vpc"
 DEPLOYMENT_REGION: "{{ region }}"
 DEPLOYMENT_SSL_CERTIFICATE_ID: 'arn:aws:acm:<REGION>:<ACCOUNT>:certificate/<GUID>'
 # DEPLOYMENT_VPC_ID: vpc-deadbeefcafe
-DEPLOYMENT_NUMBER: "{{ cluster_subnet_index|int + 1 | int }}"
+deployment_subnet_index: "{{ (CLUSTER_NUMBER - 1 |int|abs) % 2 |int }}"
+DEPLOYMENT_NUMBER: "{{ deployment_subnet_index|int + 1 | int }}"
```

Comment:
The displayed number is 1-indexed, whereas the actual data structure is 0-indexed.